File: Python/program20.py
def main():
    
    iValue = int(input("Enter number \n"))
    
    bRet = Check_bool(iValue)
    
    if(bRet == True):
        print("Even number")
    else:
        print("Odd Number")
    
def Check_bool(iNo):
    if((iNo % 2) == 0):
        return True
    else:
        return False
    

    
    
if __name__ == "__main__":
    main()

File: C++/program20.cpp

#include<iostream>
using namespace std;



int Summation(int iNo)
{
    int iCnt = 0, iSum = 0;

    for(iCnt = 0; iCnt <= iNo; iCnt++)
    {
        iSum = iSum + iCnt;   
       
    }

    return iSum;
  
}

int main()
{
    int iValue = 0, iRet = 0;

    cout<<"Enter number"<<'\n';
    cin>>iValue;

    iRet = Summation(iValue);

    cout<<"Sum is : "<<iRet<<'\n';

    return 0;
}  